r2/r2/lib/eventcollector.py

```diff
@@ -469,7 +475,7 @@ def get_context_data(self, request, context):
 
         data["domain"] = request.host
         data["user_agent"] = request.user_agent
-
+        data["referer"] = request.headers.get("Referer", None)
```

Comment:
Joe got there first, but i was gonna ask about propagating the unfortunate spelling mistakes of our ancestors here :)